src/main/java/com/eva/ui/ApplicantListPanel.java

```diff
@@ -0,0 +1,52 @@
+package com.eva.ui;
+
+
+import java.util.logging.Logger;
+
+import com.eva.commons.core.LogsCenter;
+import com.eva.model.person.applicant.Applicant;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+
+/**
+ * Panel containing the list of staffs.
+ */
+public class ApplicantListPanel extends UiPart<Region> {
+    private static final String FXML = "ApplicantListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(PersonListPanel.class);
```

Comment:
edit the logging as well. I think staff list panel should have the same problem.